stable/external-dns/templates/deployment.yaml

```
@@ -105,6 +105,9 @@ spec:
         {{- range .Values.aws.zoneTags }}
         - --aws-zone-tags={{ . }}
         {{- end }}
+        {{- if .Values.aws.preferCNAME }}
+        - --aws-prefer-cname
```

Do you need to set a value for this parameter?
```suggestion
        - --aws-prefer-cname={{ .Values.aws.preferCNAME }}
```

---

This parameter doesn't need value as it is just a flag, but the user should have the flexibility to either choose it or not, so I added it a key/value with the boolean operators in the values.yaml file.